libs/analytics/WordPressAnalytics/src/main/java/org/wordpress/android/analytics/AnalyticsTrackerNosara.java

```diff
@@ -286,6 +286,9 @@ public void track(AnalyticsTracker.Stat stat, Map<String, ?> properties) {
             case SUPPORT_OPENED_HELPSHIFT_SCREEN:
                 eventName = "support_helpshift_screen_opened";
                 break;
+            case SUPPORT_SENT_REPLY_TO_SUPPORT_MESSAGE:
+                eventName = "support_sent_reply_to_support_message";
```

Comment:
Could you please change `support_sent_reply_to_support_message` to `support_reply_to_support_message_sent` ?